sites/partners/src/listings/Aside.tsx

```diff
@@ -21,6 +21,7 @@ type AsideProps = {
   setStatus?: (status: ListingStatus) => void
   showCloseListingModal?: () => void
   showLotteryResultsDrawer?: () => void
+  showPublishModal?: () => void
```

Comment:
Seems a little weird to me to add a LM-specific prop to a component used in other places outside of LM

Reply:
I'm not sure I follow…this component is used just for the view/edit pages for Listings.

Reply:
That's my b! I saw `<aside>` but this is `<Aside>`